### PR TITLE
Create statesync.bash

### PR DESCRIPTION
**Pull request by faddat**
## Description

State sync:

Make gaia happen in about ten minutes

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/main/README.md#merging-a-pr))

### CODE REVIEW

File: contrib/statesync.bash [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# MAKE HOME FOLDER AND GET GENESIS
+gaiad init test 
+wget -O ~/.gaia/config/genesis.json https://cloudflare-ipfs.com/ipfs/Qmc54DreioPpPDUdJW6bBTYUKepmcPsscfqsfFcFmTaVig
+
+INTERVAL=1000
+
+# GET TRUST HASH AND TRUST HEIGHT
+
+LATEST_HEIGHT=$(curl -s 162.55.132.230:2011/block | jq -r .result.block.header.height);
```
Comment (zmanian) [inline contrib/statesync.bash]: Not sure if I'm on board with adding a community IP address but I would feel better about this if was behind dns

**Review by zmanian — CHANGES_REQUESTED** [review]

File: contrib/statesync.bash (hunk 1 above)
Comment (faddat) [reply contrib/statesync.bash]: neither am I :).  

I will rig up a cloudflare thing.

File: contrib/statesync.bash [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# microtick and bitcanna contributed significantly here.
+set -uxe
+
+# set environment variables
+export GOPATH=~/go
+export PATH=$PATH:~/go/bin
+
+
+# Install Gaia
+go install ./...
```
Comment (alexanderbez) [inline contrib/statesync.bash]: can we use make targets instead please?

File: contrib/statesync.bash (hunk 1 above)
Comment (alexanderbez) [reply contrib/statesync.bash]: marking as unresolved until this is updated 👍

**Review by alexanderbez — CHANGES_REQUESTED** [review]

File: contrib/statesync.bash (hunk 2 above)
Comment (faddat) [reply contrib/statesync.bash]: totally

**Comment by faddat** [comment]
We may or may not want to drop the github action.

**Review by alexanderbez — APPROVED** [review]

**Review by zmanian — APPROVED** [review]

**Pull request merged**